jbpm-runtime-manager/src/test/java/org/jbpm/runtime/manager/impl/factory/InMemorySessionFactoryLeakTest.java

```diff
@@ -79,14 +83,18 @@ public InMemorySessionFactoryLeakTest(String strategy) {
     public TestName testName = new TestName();
     
     @Before
-    public void setup() {
+    public void setup() throws NamingException {
         TestUtil.cleanupSingletonSessionId();
         Properties properties= new Properties();
         properties.setProperty("mary", "HR");
         properties.setProperty("john", "HR");
         userGroupCallback = new JBossUserGroupCallbackImpl(properties);
         
         createRuntimeManager();
+
+        new InitialContext().rebind("java:comp/UserTransaction", com.arjuna.ats.jta.UserTransaction.userTransaction());
+        new InitialContext().rebind("java:comp/TransactionManager", TransactionManager.transactionManager());
+        new InitialContext().rebind("java:comp/TransactionSynchronizationRegistry", new com.arjuna.ats.internal.jta.transaction.arjunacore.TransactionSynchronizationRegistryImple());
```

Comment:
great - it was a new one added recently and seemed to need the transaction manager available for some reason (or maybe it was TSR)